_data/students.yml

```diff
@@ -1,3 +1,9 @@
+ -name: Chirag Kumar
```

Comment:
Please use the exact format as it is given in the readme.md. There should be a space after `-`